api/config/mongoid.yml

```diff
@@ -13,10 +13,10 @@ development:
       # Provides the hosts the default client can connect to. Must be an array
       # of host:port pairs. (required)
       hosts:
-        - "localhost:<%= ENV['MONGODB_PORT'] || raise('Missing required environment variable: MONGODB_PORT') %>"
+        localhost: <%= ENV['MONGODB_PORT'] %>
       options:
-        user: <%= ENV["MONGODB_USERNAME"] || raise("Missing required environment variable: MONGODB_USERNAME") %>
```

Comment:
from experience: better to have this check. Maybe not in config yml file, but better to check env variables and raise error/log errors if they are missing